constraints-dev.txt

```diff
@@ -1,4 +1,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 torch<2.7.0
-vllm<0.9.0
+vllm<0.9.0
```

Comment:
Presumably changing this line because it has the string "0.9.0" here, and the editor altered the newline on this file.